data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1098,7 +1098,7 @@ spec:
                     description: ClusterOSImage is either a URL with `http(s)` or `file` scheme to override the default OS image for cluster nodes, or an existing Glance image name.
                     type: string
                   computeFlavor:
```

Comment:
Can you remove this property from [docs/user/openstack/customization.md](https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md) too? It's currently marked as required...

Reply:
make sure @maxwelldb  is aware

Reply:
Is the short of this that `flavor` in `platform.openstack.defaultMachinePlatform` replaces `computeFlavor` for 4.7+?

Reply:
platform.openstack.computeFlavor is now depricated, but still supported.
If you want to set a compute flavor to be used by default for all machine sets, you will now need to set it in platform.openstack.defaultMachinePlatform.flavorName. Otherwise, you should set the flavor in each machine pool